smpe/bld/smpe.sh

```diff
@@ -272,6 +274,12 @@ _stopAt smpe-service.sh $debug -c $YAML $opts
 _cmd $here/smpe-service.sh $debug -c $YAML $opts
 # result (final): $ship                      # zip with sysmod & readme
 
+# create psi
```

Comment:
the whole code block should be wrapped in the if
`# create psi`
`if test -n "$psi"`
`then`
`  opts=""`
`  _stopAt prepare-PSI.sh $debug -c $YAML $opts`
`  _cmd $here/psi/bld/prepare-PSI.sh $debug -c $YAML $opts`
  
`  # TODO create PSI`
`fi    # $psi`

Reply:
Updated based on feedback